src/app/integrations/create-page/create-page.component.ts

```diff
@@ -42,8 +42,30 @@ export class IntegrationsCreatePage implements OnInit, OnDestroy {
     this.router.navigate(['/integrations']);
   }
 
+  showNext() {
+    return !(this.getCurrentChild() === 'connection-configure' && this.position === 1);
+  }
+
+  showFinish() {
+    return this.getCurrentChild() === 'connection-configure' && this.position === 1;
+  }
+
+  canFinish() {
```

Comment:
I personally like to have functions in ABC order, not being sarcastic here, I'm truly that OCD. Because when files get really long (though they shouldn't or should be broken up), it makes it a lot easier to find what you need assuming you remember the name of the function. Which means, really, that documentation and updating the style guide is probably more important than anything. In other words, please ignore this comment.